### PR TITLE
actions.py now prints logs; basic functionality

### PR DESCRIPTION
**Pull request by Joecakes4u**
progress on #244

### CODE REVIEW

File: vent/api/actions.py [hunk 1]
```diff
@@ -177,7 +177,16 @@ def system_commands():
...
 
    @staticmethod
-   def logs():
+   def logs(grep_list=None):
+       # generically filter logs stored in log containers
+       cores = ["core-aaa-syslog", "core-aaa-redis", "core-aaa-rabbitmq", "core-rmq-es-connector", "vent-management"]
+       if grep_list:
+           for expression in grep_list:
+               for core in cores:
+                   os.system("docker logs "+core+" | grep "+expression+" | tee")
```
Comment (cglewis) [inline vent/api/actions.py]: I added the `docker-py` library to work natively with the vent package, so we should use that for docker commands to make it less brittle in the future.

check this out for how to set up the environment:

https://github.com/CyberReboot/vent/blob/experimental/vent/menu.py#L10

Comment (Joecakes4u) [inline vent/api/actions.py]: Clever. I'll make those changes.

**Comment by cglewis** [comment]
regarding the docker error in circleci, we might need to try this:

```
You can side load docker by adding the follwoing. Remember to remove docker from services list if you have it.

machine:
  post:
    - curl -fsSLO https://get.docker.com/builds/Linux/x86_64/docker-1.13.1.tgz && sudo tar --strip-components=1 -xvzf docker-1.13.1.tgz -C /usr/local/bin
    - sudo container=yes docker daemon: { background: true }
```

and then update the `tests/requirements.txt` to have a newer version of docker-py

**Comment by cglewis** [comment]
you'll have to remove docker from services:

https://github.com/Joecakes4u/vent/blob/26f98478c5d226792dc01159032e2f79ba3d6f83/circle.yml#L5

Also, note that the section is `machine:` not `test:`

File: vent/api/actions.py [hunk 2]
```diff
@@ -176,8 +179,16 @@ def system_commands():
        # restart, shutdown, upgrade, etc.
        return
 
-   @staticmethod
-   def logs():
+   def logs(grep_list=["core","vent-management"]):
```
Comment (cglewis) [inline vent/api/actions.py]: this needs to be tested, I think you're missing `self` as an argument, at a minimum.

**Comment by cglewis** [comment]
@Joecakes4u I made several updates that I think fixed the Docker problems you were running into.  Now that there are conflicts with your PR, you'll probably want to just keep the changes to the `logs` function that filled in, and get rid of the rest, pulling into the new changes.

**Comment by Joecakes4u** [comment]
Should `vendor` be added to .gitignore?

**Comment by Joecakes4u** [comment]
Each stub in actions.py is three spaces from the left edge of the screen, but the first line inside the scope of a stub is four spaces from the function definition. Is this intended, or should I adjust this?

**Comment by cglewis** [comment]
no, vendor shouldn't be in .gitignore, suplemon should be a gitmodule, which I realize now is outdated since I changed the path - i think `.gitmodule needs to be updated.

the 3 spaces is my mistake, everything should be 4, please feel free to fix.
